coderd/apikey.go

```diff
@@ -354,7 +354,7 @@ func (api *API) tokenConfig(rw http.ResponseWriter, r *http.Request) {
 	httpapi.Write(
 		r.Context(), rw, http.StatusOK,
 		codersdk.TokenConfig{
-			MaxTokenLifetime: values.MaxTokenLifetime.Value(),
+			MaxTokenLifetime: values.Sessions.MaximumTokenDuration.Value(),
```

Comment:
Instead of grouping this config value with sessions, should we leave it as is before? The naming discrepancy here is confusing

Reply:
I really want to include it in the grouping because the grouping highlights how confusing everything is imo.

Essentially we have these different things, "token", vs "cli auth", vs "browser api key". They are all "Sessions", but we do not have good documentation what the differences are.

If I omit `MaximumTokenDuration` from the grouping, it's continuing the scattered concept

Reply:
I really want to group all concepts of a "Session" from the configuration perspective.

Essentially we have these different things, "token", vs "cli auth", vs "browser api key". They are all "Sessions", but we do not have good documentation what the differences are.

If I omit `MaximumTokenDuration` from the grouping, it's continuing the scattered concept. 

I agree the current names are confusing, and I actually had this PR fixing the names, but the `createAPIKey` function doesn't lend itself well to it at present.

I could rename `MaximumTokenDuration` -> `MaxTokenLifetime`. It just feels werid to say "MaxLifetime" as we allow refreshing _some_ sessions, but not others. I like the word `Duration` more personally.